### DISCUSSION TITLE
Help Needed - "Error loading page: Domain: undefined Error Code: -9 Description: NEVER_TOO_MANY_REDIRECTS"

### DISCUSSION THREAD

**Discussion by Kawan-idres** [vercel/next.js#61768] [Help]
### Summary

Hi Next.js community,

I hope you're all doing well. I'm reaching out because I've encountered an issue with my Next.js 14 Pages Directory  eCommerce project that's causing random crashes for some of my users. While the project works fine for me, some users are reporting an error message stating:

"Error loading page: Domain: undefined Error Code: -9 Description: NEVER_TOO_MANY_REDIRECTS"

This error seems to be related to too many redirects, but I'm having trouble pinpointing the exact cause as it doesn't consistently occur for all users. I've checked my routing configurations, but everything seems to be in order.

Has anyone else experienced a similar issue before? If so, could you please provide some insights or suggestions on how to troubleshoot and resolve this issue? Any help would be greatly appreciated.

Thank you in advance for your assistance!
<img width="394" alt="Screenshot 2024-02-07 at 5 28 49 PM" src="https://github.com/vercel/next.js/assets/139050865/dc204e7d-2af6-4dc6-b58d-8e556fd8ce6f">

### Example

https://lokal.lezzoodevs.com/

**Comment by farhanali30799-sudo** [comment]
Error

**Comment by farhanali30799-sudo** [comment]
Code
Issues
2.3k
Help Needed - "Error loading page: Domain: undefined Error Code: -9 Description: NEVER_TOO_MANY_REDIRECTS" #61768
Unanswered
Kawan-idres asked this question in Help

**Reply by farhanali30799-sudo** [reply]
Nice
